.eslintrc.json

```diff
@@ -44,6 +44,7 @@
       "always"
     ],
     "no-useless-return": 1,
-    "class-methods-use-this": 0
+    "class-methods-use-this": 0,
+    "no-lonely-if": 0
```

Comment:
@MutegekiHenry  could you kindly explain What this is doing?

Reply:
it handles es lint error for a scenario where you use an if without an else yet es-lint expected the else. No biggie though

Reply:
Aaahhh makes sense.